tests/unit/evaluators/EvalTestSetup.hpp

```diff
@@ -92,11 +92,10 @@ createTestLayoutAndBasis(Teuchos::RCP<Albany::Layouts> &dl,
 
 }
 
-void createTestMapsAndWorksetConns(
+Albany::WorksetConn createTestMapsAndWorksetConns(
```

Comment:
Why did you choose to return `wsLocalElNodeEqID`. I don't understand why `wsLocalElNodeEqID` should be handled differently than `wsGlobalElNodeEqID`.